### PR TITLE
Use nightly toolchain for rustfmt to get access to merge_imports

### PR DESCRIPTION
**Pull request by thomaseizinger**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,18 +1,29 @@
 RUSTUP = rustup
+
 TOOLCHAIN = $(shell cat rust-toolchain)
-CARGO = $(RUSTUP) run --install $(TOOLCHAIN) cargo --color always
+CARGO = $(RUSTUP) run $(TOOLCHAIN) --install cargo --color always
```
Comment (D4nte) [inline Makefile]: 📣  The original line was correct:
```
▶ rustup run --help
rustup-run 
Run a command with an environment configured for a given toolchain

USAGE:
    rustup run [FLAGS] <toolchain> <command>...
```

File: Makefile [hunk 2]
```diff
@@ -1,18 +1,29 @@
...
+
+NIGHTLY_TOOLCHAIN = "nightly-2019-07-31"
+CARGO_NIGHTLY = $(RUSTUP) run $(NIGHTLY_TOOLCHAIN) --install cargo --color always
```
Comment (D4nte) [inline Makefile]:
```suggestion
CARGO_NIGHTLY = $(RUSTUP) run --install $(NIGHTLY_TOOLCHAIN) cargo --color always
```
See previous comment.

File: Makefile [hunk 3]
```diff
@@ -1,18 +1,29 @@
...
 
 build: build_debug
 
+install_rust:
```
Comment (D4nte) [inline Makefile]: 📣  This is not needed, `--install` does it for you.

Comment (thomaseizinger) [reply Makefile]: https://circleci.com/gh/comit-network/create-comit-app/711 :(

File: Makefile (hunk 1 above)
Comment (thomaseizinger) [reply Makefile]: True, I misread the help text of the command :)

File: Makefile (hunk 3 above)
Comment (thomaseizinger) [reply Makefile]: `--install` only works for the subcommand `run`. It doesn't work for the sub command `components` which we execute first in order to see if we have all the components installed ...

**Comment by thomaseizinger** [comment]
@D4nte all feedback addressed and merge conflicts are fixed :)

File: Makefile (hunk 3 above)
Comment (D4nte) [reply Makefile]: dang!

**Pull request merged**